js/interface.js

```diff
@@ -150,4 +158,8 @@ Fliplet().then(function() {
 
     initDataSourceProvider();
   });
+
+  $('[data-toggle="tooltip"]').tooltip();
+
```

Comment:
Line 163, extra line needed ?

Reply:
Not needed. I did the changes. I observe that sso-saml2 widget not capturing eslint properly.

Reply:
@amolsuryawanshi47  Please explain the reason behinde ESLint not working for SSO Saml 2.

Reply:
there is no eslint package in package.son or widget.json file